cmd/agent/app/check.go

```diff
@@ -131,6 +131,10 @@ var checkCmd = &cobra.Command{
 			fmt.Println(string(checkStatus))
 		}
 
+		if checkRate == false {
+			color.Yellow("Currently only displaying Gauge metrics. You can run the command with --check-rate to see any other metric types if available.")
```

Comment:
@nmuesch: This message is a bit misleading: one check run is enough for other types of check-submitted metrics than just `Gauge`. For instance metrics submitted with `Sender.Count`/`self.count`, `Sender.Histogram`/`self.histogram` are shown without requiring `--check-rate`.

We should probably reword this to mention that the command is "not displaying metrics submitted with `Rate` and `MonotonicCount`", instead of "only displaying Gauge metrics".

Thoughts?

Reply:
Good point. Best option would be avoid referring to specific metric types, to avoid having to change this string whenever we change something on the aggregator side (unlikely but still).
Something like:
> Check has run only once, if some metrics are missing you can try again with --check-rate to see any other metric if available.

Reply:
Good call @olivielpeau @masci I've updated the wording to remove specific mention to gauges here - https://github.com/DataDog/datadog-agent/pull/1460